example.py

```diff
@@ -31,74 +26,60 @@
 type_column = 'colType'
 model_weights_path = 'torontobooks_unigrams.bin'
 
-open_d3m_file = 'data/open_d3m_data.csv'
-closed_d3m_file = 'data/closed_d3m_data.csv'
+open_d3m_file = '~/data/open_d3m_unembed_data.csv'
+closed_d3m_file = '~/data/closed_d3m_unembed_data.csv'
 
-files = [open_d3m_file]
+#files = [open_d3m_file]
 #files = [open_d3m_file, closed_d3m_file]
-#files = [closed_d3m_file, open_d3m_file]
+files = [closed_d3m_file, open_d3m_file]
+
+def fit(model_class, xtrain, ytrain, xtest, ytest, isBalanced, dump=False):
+    print('evaluating model: {}'.format(model_class.__name__))
+    model = model_class()
+    print('fitting model...')
+    model.fit(xtrain, ytrain)
+
+    if (isBalanced and dump):
+        filename = 'RF_public_model.sav'
+        pickle.dump(model, open(filename, 'wb'))
+
+    yhat = model.predict(xtest)
+    accuracy = accuracy_score(ytest, yhat)
+    f1_micro = f1_score(ytest, yhat, average='micro')
+    f1_macro = f1_score(ytest, yhat, average='macro')
+    f1_weighted = f1_score(ytest, yhat, average='weighted')
+
+    return {'data_collection': data_collection, 'classifier': model_class.__name__, 'balanced': isBalanced, 'accuracy_score': accuracy, 'f1_score_micro': f1_micro, 'f1_score_macro': f1_macro, 'f1_score_weighted': f1_weighted}
+
+def split_X_and_y(data: pd.DataFrame):
+    X = data.drop(['datasetName', type_column], axis=1)
+    y = data[type_column]
+    return X, y
```

Comment:
the name split has a connotation of train/test split. is there another name we could use? `extract`? `separate`?